BackEnd/src/main/java/com/exadel/carpoolfree/controller/DriveController.java

```diff
@@ -0,0 +1,49 @@
+package com.exadel.carpoolfree.controller;
+
+
+import com.exadel.carpoolfree.model.Drive;
+import com.exadel.carpoolfree.repository.DriveRepository;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("api/drive")
+public class DriveController {
+    private final DriveRepository driveRepository;
+
+    public DriveController(DriveRepository driveRepository) {
+        this.driveRepository = driveRepository;
+    }
+
+    //Is it need to return absolutely all drives?
```

Comment:
If we display drives history then we need all as well

Reply:
done1

Reply:
done